src/forms/STEP_PROPS.js

```diff
@@ -1,10 +1,10 @@
-import { CurrentIncomeStep } from './forms/CurrentIncome';
-import { CurrentExpensesStep } from './forms/CurrentExpenses';
-import { PredictionsStep } from './forms/Predictions';
-import { HouseholdStep } from './forms/Household';
-import { CurrentBenefitsStep } from './forms/CurrentBenefits';
+import { CurrentIncomeStep } from './CurrentIncome';
+import { CurrentExpensesStep } from './CurrentExpenses';
+import { PredictionsStep } from './Predictions';
+import { HouseholdStep } from './Household';
+import { CurrentBenefitsStep } from './CurrentBenefits';
 
-export const STEPS = [
+export const STEP_PROPS = [
```

Comment:
I would avoid using "props" here since it has a specific meaning in React-land, and suggests these are meant to be passed as component props. Why do you feel `STEPS` is insufficient?

Reply:
Fair point. `STEPS` feels like a list of components to me, but it's more than that. Any suggestions? `STEP_VALS`? `STEP_PARTS`?